### ISSUE TITLE
Accessibility bug: missing alt tag from iOS WordPress app when field left blank for decorative images [Type] Bug

### ISSUE THREAD

**Issue by baeolophus** [wordpress-mobile/WordPress-iOS#21162]
### Expected behavior
Leaving alt text blank should result in alt tag containing "" (empty) for decorative images from any app or dashboard.

### Actual behavior
Leaving alt text blank results in NO alt tag when posted from WordPress iOS app.  Leaving alt text blank from browser dashboard works correctly providing alt="".  Leaving alt text blank from Android WordPress app correctly provides alt="".

### Steps to reproduce the behavior
Added images to blog posts from the browser dashboard with and without alt text; added images to a blog post from the iOS app with and without alt text; checked images from older posts from before my use of the iOS app

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
